Gaijinizer patch/Map041.txt

```diff
@@ -1075,18 +1147,19 @@ Mm ......... It's delicious! !#MTLed
 ホントに人に振る舞うの初めてなの！？
 ぜってー違うでしょ！！この味加減…！！！
 > CONTEXT: Map041/events/5/pages/1/491/Dialogue/kazuya-a
-I mean this!!It is the first time to truly act
-on people! ? All right-not different!!This
-taste is ...!!!#MTLed
+I mean, honestly!! Is this really the first time
+you've cooked for someone?! There's nooo way!! Not
+if you can make something that tastes like this!!!#Ron
```

Comment:
Adding some punctuation can help make this flow....
>_**I mean, honestly...!! Is this reaallly the first time
you've cooked for someone?! There's no way...!!
Not if you can make something that tastes like this!!**_